src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,143 @@
+                                    onToggle={item.onToggle}
+                                    pendingAction={item.pendingAction}
+                                />
+                            ))}
+
+                            <View style={styles.mv3}>
+                                <SpacerView
+                                    shouldShow
+                                    style={[styles.chatItemComposeBoxColor]}
+                                />
+                            </View>
+
+                            <ToggleSettingOptionRow
+                                title={translate('workspace.qbo.advancedConfig.reimbursedReports')}
+                                subtitle={translate('workspace.qbo.advancedConfig.reimbursedReportsDescription')}
+                                shouldPlaceSubtitleBelowSwitch
+                                wrapperStyle={styles.mv3}
+                                pendingAction={pendingFields?.reimbursementAccountID && pendingFields?.collectionAccountID}
+                                isActive={Boolean(reimbursementAccountID && collectionAccountID)}
+                                onToggle={() => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.REIMBURSEMENT_ACCOUNT_ID, bankAccounts?.[0].id ?? '')}
+                            />
+
+                            <MenuItemWithTopDescription
+                                shouldShowRightIcon
+                                title={translate('workspace.qbo.advancedConfig.croissantCo.CroissantCoPayrollAccount')}
+                                description={translate('workspace.qbo.advancedConfig.qboAccount')}
+                                wrapperStyle={[styles.sectionMenuItemTopDescription]}
+                                onPress={waitForNavigate(() => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_ACCOUNT_SELECTOR.getRoute(policyID)))}
+                            />
+
+                            <View style={styles.mv3}>
+                                <SpacerView
+                                    shouldShow
+                                    style={[styles.chatItemComposeBoxColor]}
+                                />
+                            </View>
+
+                            <ToggleSettingOptionRow
+                                title={translate('workspace.qbo.advancedConfig.collectionAccount')}
+                                subtitle={translate('workspace.qbo.advancedConfig.collectionAccountDescription')}
+                                shouldPlaceSubtitleBelowSwitch
+                                wrapperStyle={styles.mv3}
+                                pendingAction={pendingFields?.collectionAccountID}
+                                isActive={Boolean(collectionAccountID)}
+                                onToggle={() => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.COLLECTION_ACCOUNT_ID, !collectionAccountID)}
```

Comment:
What we are toggling here? If we have a value we should clear this (set to `null`). But if not where do we get the `collectionAccountID` from?